communications/README.md

```diff
@@ -0,0 +1,11 @@
+# Communications
+
+## Async written first
+
+- If you want to discuss an idea (e.g. new or existing product feature, update a marketing page), comment on or create a new story/epic in Clubhouse. @-mention specific individuals and ask them specific questions. See [Product Development](../product-development/README.md).
+- Ensure your [Clubhouse notifications](https://help.clubhouse.io/hc/en-us/sections/201543416-Notifications) are properly configured. Respond to Tilters within a few days with an answer, or at least say you don't have time currently to do so.
+- If you want to discuss a process change, consider the smallest reasonable change and create a GitHub PR in this repo as a proposal. @-mention and assign specific individuals for review.
+- Only when you have exhausted discussions in Clubhouse and/or GitHub, consider scheduling a meeting with a clear agenda to determine next steps.
+- When using Slack, prefer using public channels over DMs. @-mention specific individuals with a question if you want a response. Prefer using threads.
```

Comment:
I might reframe this slightly. How would you feel about something like: 

> Suppose you have an idea for a feature, but you're not sure where it belongs. Or you see weird behavior in Tilt, and are not sure is a bug. Please ask in Slack! Ask in a shared channel, rather than a private DM.

> If you get confirmation that this is a common issue that people want to discuss more, move the more substantial discussion to a product-focused venue, like Github issues or a Clubhouse story

Reply:
I actually rather this _not_ be in Slack, and that people go to Clubhouse directly. I have lots of reasons for this, that not everyone might agree with now. So I'll just delete this line and we can re-visit again.